### DISCUSSION TITLE
Dispatch To not working

### DISCUSSION THREAD

**Discussion by ylcnztrk** [livewire/livewire#8111] [1. Help]
Hi,
"Dispatch to" command does not work.
Even though it works within the same component, a different component cannot be included.

Example;

_**Example Button Command**_ 
```blade
<button wire:click="example">example</button>
``` 

**App/Livewire/Dashboard/Sections/Applications/Create.php** 
```php
    public function example() {
        $this->dispatch('example')->to(\App\Livewire\Dashboard\Sections\Customers\Table::class);
    }
``` 
**App/Livewire/Dashboard/Sections/Customers/Table.php**
```php
use Livewire\Attributes\On;

    #[On('example')]
    public function test() {
        dd('example test');
    }
```

**Comment by romansh** [comment]
Did you add 
use Livewire\Attributes\On; 
in App/Livewire/Dashboard/Sections/Customers/Table.php ?

**Reply by ylcnztrk** [reply]
Yes, I'm already using that.

**Reply by romansh** [reply]
For some reason I hadn't spotted it in your code. :)

Have you tried 

public function example() {
        dd('Yes!');
    }
    
   Maybe there is some issue with routes or something else. I just ran similar code with dispatch to and it was working.

**Comment by danie-ramdhani** [comment]
I made example based on your case:
https://wirebox.app/b/yrdqq

maybe the `Table` component is not included in your main component/layout?
